shared/actions/fs/index.tsx

```diff
@@ -1147,6 +1025,54 @@ const setTlfsAsUnloadedWhenKbfsDaemonDisconnects = state =>
 const setDebugLevel = (_, action: FsGen.SetDebugLevelPayload) =>
   RPCTypes.SimpleFSSimpleFSSetDebugLevelRpcPromise({level: action.payload.level})
 
+const subscriptionDeduplicateIntervalSecond = 1
+
+const subscribePath = (_, action: FsGen.SubscribePathPayload) =>
+  RPCTypes.SimpleFSSimpleFSSubscribePathRpcPromise({
+    deduplicateIntervalSecond: subscriptionDeduplicateIntervalSecond,
+    kbfsPath: Types.pathToString(action.payload.path),
+    subscriptionID: action.payload.subscriptionID,
+    topic: action.payload.topic,
+  })
+
+const subscribeNonPath = (_, action: FsGen.SubscribeNonPathPayload) =>
+  RPCTypes.SimpleFSSimpleFSSubscribeNonPathRpcPromise({
+    deduplicateIntervalSecond: subscriptionDeduplicateIntervalSecond,
+    subscriptionID: action.payload.subscriptionID,
+    topic: action.payload.topic,
+  })
+
+const unsubscribe = (_, action: FsGen.UnsubscribePayload) =>
+  RPCTypes.SimpleFSSimpleFSUnsubscribeRpcPromise({
+    subscriptionID: action.payload.subscriptionID,
+  }).catch(() => {})
+
+const onPathChange = (_, action: EngineGen.Keybase1NotifyFSFSSubscriptionNotifyPathPayload) => {
+  const {path, topic} = action.payload.params
+  switch (topic) {
+    case RPCTypes.PathSubscriptionTopic.children:
+      return FsGen.createFolderListLoad({path: Types.stringToPath(path)})
+    case RPCTypes.PathSubscriptionTopic.stat:
+      return FsGen.createLoadPathMetadata({path: Types.stringToPath(path)})
+    default:
+      Flow.ifFlowComplainsAboutThisFunctionYouHaventHandledAllCasesInASwitch(topic)
```

Comment:
i believe we no longer need this construct. due to the 'not all paths return a value' if you remove the default it should be fine and if you commented out one handler it'd complain